packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2539,40 +2537,40 @@ class RectangularSliderValueIndicatorShape extends SliderComponentShape {
   Size getPreferredSize(
      bool isEnabled,
      bool isDiscrete, {
-     @required TextPainter labelPainter,
-     @required double textScaleFactor,
+     TextPainter? labelPainter,
+     double? textScaleFactor,
```

Comment:
same

Reply:
Those 2 parameters can not be required because `getPreferredSize` overrides `SliderComponentShape.getPreferredSize(bool isEnabled, bool isDiscrete)` that does not define those optional parameters.

Reply:
Oh wow, interesting.